### PR TITLE
Add quorumThresholdPercentage to MainTaskSender

### LINKED ISSUE #61: feat (aggregator): allow users to configure quorum stake % when submitting new task
- Need to change aggregator to use configured threshold before submiting response

### PR DESCRIPTION
**Pull request by isoprophlex**
Added option to customize quorumThresholdPercentage via config-file in MainTaskSender before Task is created in:
`	task := types.NewTask(provingSystem, proofFile, publicInputFile, verificationKeyFile, uint8(quorumThresholdPercentage))
`
I also added a print to check if the percentage was set correctly before sending the task.
Captura de pantalla 2024-05-02 a la(s) 17 12 36
Captura de pantalla 2024-05-02 a la(s) 17 17 46

### CODE REVIEW

File: task_sender/cmd/main.go [hunk 1]
```diff
@@ -127,9 +129,12 @@ func taskSenderMain(c *cli.Context) error {
 	}
 
 	taskSender := pkg.NewTaskSender(avsWriter)
-	quorumThresholdPercentage := uint8(100) // TODO: add this to the configuration
-	task := types.NewTask(provingSystem, proofFile, publicInputFile, verificationKeyFile, quorumThresholdPercentage)
-
+	quorumThresholdPercentage, ok, err := getQuorumThresholdPercentage(c.String(config.ConfigFileFlag.Name)) // Read from config
```
Comment (NicolasRampoldi) [inline task_sender/cmd/main.go]: Missing error check.

File: task_sender/cmd/main.go [hunk 2]
```diff
@@ -127,9 +129,12 @@ func taskSenderMain(c *cli.Context) error {
...
+	task := types.NewTask(provingSystem, proofFile, publicInputFile, verificationKeyFile, uint8(quorumThresholdPercentage))
+	if !ok {
+		return errors.New("quorum threshold percentage not found or not an int in config file")
+	}
+	println(quorumThresholdPercentage)
```
Comment (NicolasRampoldi) [inline task_sender/cmd/main.go]: Remove print

**Review by NicolasRampoldi — CHANGES_REQUESTED** [review]

File: Makefile [hunk 3]
```diff
@@ -97,6 +97,7 @@ send-plonk-proof: ## Send a PLONK proof using the task sender
 		--public-input task_sender/test_examples/plonk_pub_input.pub \
 		--verification-key task_sender/test_examples/plonk.vk \
 		--config config-files/config.yaml \
+		--q 25 \
```
Comment (NicolasRampoldi) [inline Makefile]: Use something more descriptive instead of `q`.

File: config-files/config.yaml [hunk 4]
```diff
@@ -37,3 +37,7 @@ private_key_store_path: config-files/anvil.ecdsa.key.json
 bls_private_key_store_path: config-files/anvil.bls.key.json
 signer_type: local_keystore
 chain_id: 31337
+
+## Task Sender Configurations
+task_sender:
+  quorum_threshold_percentage: 92
```
Comment (NicolasRampoldi) [inline config-files/config.yaml]: Remove if it is no longer being used

**Review by NicolasRampoldi — CHANGES_REQUESTED** [review]

**Review by NicolasRampoldi — APPROVED** [review]

**Review by entropidelic — APPROVED** [review]

**Pull request merged**